test/Sentry.Tests/Internals/SimpleStackTraceFactoryTests.cs

```diff
@@ -0,0 +1,42 @@
+// ReSharper disable once CheckNamespace
+// Stack trace filters out Sentry frames by namespace
+namespace Other.Tests.Internals;
+
+#if PLATFORM_NEUTRAL
+
+public class SimpleStackTraceFactoryTests
+{
+    [Fact]
+    public Task MethodGeneric()
+    {
+        // Arrange
+        const int i = 5;
+        var exception = Record.Exception(() => GenericMethodThatThrows(i));
+
+        var options = new SentryOptions
+        {
+            AttachStacktrace = true
```

Comment:
question: I believe `AttachStacktrace = true` currently has no effect ... should it have any?